src/containers/Layout/Layout.tsx

```diff
@@ -104,7 +107,7 @@ const Layout = () => {
     if (!clientId) return null;
 
     return isLoggedIn ? (
-      <UserMenu showPaymentsItem={accessModel !== 'AVOD'} />
+      <UserMenu showPaymentsItem={accessModel !== ACCESS_MODEL.AVOD} />
```

Comment:
```suggestion
      <UserMenu showPaymentsItem={accessModel !== ACCESS_MODEL.AVOD} />
```
```suggestion
      <UserMenu showPaymentsItem={accessModel === ACCESS_MODEL.SVOD} />
```

Reply:
Looks like billing history can still be there when we buy a TVOD item.